managed/services/platform/platform.go

```diff
@@ -37,7 +38,10 @@ import (
 
 const rollbackFailed = "Failed to rollback:"
 
-var errGetSSODetailsFailed = status.Error(codes.Aborted, "Failed to fetch SSO details.")
+var (
+	errGetSSODetailsFailed = status.Error(codes.Aborted, "Failed to fetch SSO details.")
+	errProxyNotSupported   = status.Error(codes.Aborted, "PMM Platform connection does not support proxy.")
```

Comment:
added this error since the ticket asks for a more descritptive error instead of Internal Server https://jira.percona.com/browse/PMM-11421